experimental/packages/otlp-grpc-exporter-base/src/create-service-client-constructor.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as grpc from '@grpc/grpc-js';
+
+/**
+ * Creates a unary service client constructor that, when instantiated, does not serialize/deserialize anything.
+ * Allows for passing in {@link Buffer} directly, serialization can be handled via protobufjs or custom implementations.
+ *
+ * @param path service path
+ * @param name service name
+ */
+export function createServiceClientConstructor(
```

Comment:
Here we actually need to import `@grpc/gprc-js` to create that constructor, that's why it's in a separate file. This way we can lazy-load this file instead of having it all in the exporter transport file (which is safe to require at any time).